volatility3/framework/plugins/renderers/parquet_renderer.py

```diff
@@ -0,0 +1,222 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+import datetime
+import logging
+import sys
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+    Tuple,
+    TextIO,
+)
+from volatility3.framework import interfaces, renderers
+from volatility3.framework.renderers import format_hints
+from volatility3.cli.text_renderer import (
+    CLIRenderer,
+    display_disassembly,
+    LayerDataRenderer,
+)
```

Comment:
```suggestion
from volatility3.cli import text_renderer
```

The [volatility coding style](https://github.com/volatilityfoundation/volatility3/blob/develop/CODING_STYLE.md#imports) is to import only modules, so that people don't import a class directly from a module where it had been imported itself.  Please could you update these.  We allow it for typing because they're often repeated and very common.